### PR TITLE
Dkryvchuk/grok connect

### PR DESCRIPTION
**Pull request by dionissqq**

### CODE REVIEW

File: connectors/grok_connect/src/main/java/grok_connect/handlers/SessionHandler.java [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+    byte[] bytes;
+
+    QueryManager qm;
+
+    int queryMessages = 0;
+    static String okResponse = "DATAFRAME PART OK";
+    static String endMessage = "the end";
+    static String sizeRecievedMessage = "DATAFRAME PART SIZE RECEIVED";
+    static String logRecievedMessage = "LOG RECIEVED";
```
Comment (alex-aprm) [inline connectors/grok_connect/src/main/java/grok_connect/handlers/SessionHandler.java:33]: A typo

File: connectors/grok_connect/src/main/java/grok_connect/GrokConnect.java [hunk 2]
```diff
@@ -132,21 +137,21 @@ private static void connectorsModule() {
...
-                DataFrame result = provider.queryTable(connection, query);
-                buildResponse(response, result.toByteArray());
-            } catch (Throwable ex) {
-                buildExceptionResponse(response, printError(ex));
-            }
-
-            logMemory();
-            return response;
-        });
+        // post("/query_table", (request, response) -> {
```
Comment (LaykDimon) [inline connectors/grok_connect/src/main/java/grok_connect/GrokConnect.java:140]: do we really need this comment?

File: connectors/grok_connect/src/main/java/grok_connect/handlers/SessionHandler.java [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    public void onError(Throwable err) throws Throwable {
+        session.getRemote().sendString(socketErrorMessage(err));
+        session.close();
+        qm.closeConnection();
+    }
+
+    public void onMessage(String message) throws Throwable {
+        if (SettingsManager.getInstance().settings != null) {
+            try {
+                if (message.startsWith("QUERY")) {
```
Comment (alex-aprm) [inline connectors/grok_connect/src/main/java/grok_connect/handlers/SessionHandler.java:50]: convert to const

File: connectors/grok_connect/src/main/java/grok_connect/handlers/SessionManager.java [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+import org.eclipse.jetty.websocket.api.Session;
+
+public class SessionManager {
+    static HashMap<Session, SessionHandler> sessions = new HashMap<Session, SessionHandler>();
+
+    static void add(Session s) throws IOException {
+        
+        sessions.put(s, new SessionHandler(s));
+        s.getRemote().sendString("CONNECTED");
```
Comment (alex-aprm) [inline connectors/grok_connect/src/main/java/grok_connect/handlers/SessionManager.java:14]: convert to const

**Comment by dionissqq** [comment]
merged from another branch dkryvchuk/grok_connect2

**Pull request closed without merging**